build.gradle

```diff
@@ -20,10 +20,12 @@ repositories {
 
 dependencies {
     compile name: 'touch-image-view'
+    compile 'com.android.support:support-v4:22.2.1'
     compile project(':owncloud-android-library')
     compile 'com.android.support:support-v4:22.2.1'
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile 'com.android.support:appcompat-v7:22.2.1'
+    compile 'com.getbase:floatingactionbutton:1.10.1'
```

Comment:
.. So we could for example also move on to this lib: https://github.com/yavski/fab-speed-dial which needs the 23.1.1 lib (might be back-portable if even necessary) because it implements the speed dial with the Support Lib FAB implementation. But as I said before we would need to use some lib for the speed dial or someone would have to implement it within this app.

So how about I integrate the needed lilb classes into our third party code folder? Would that be feasable?